.github/workflows/integration_tests_manual.yml

```diff
@@ -1,10 +1,10 @@
-name: Integration Tests
+name: Integration Test - Manual Scans
```

Comment:
This name change will require changing the `required` check in the repo branch/ruleset settings or prs won't be and to be merged